packages/flutter_test/lib/src/buffer_matcher.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:typed_data';
+
+import 'package:test_api/src/frontend/async_matcher.dart'; // ignore: implementation_imports
+// ignore: deprecated_member_use
+import 'package:test_api/test_api.dart' show Description, TestFailure;
+
+import 'goldens.dart';
+
+/// Matcher created by [bufferMatchesGoldenFile].
+class _BufferGoldenMatcher extends AsyncMatcher {
+  /// Creates an instance of [BufferGoldenMatcher]. Called by [bufferMatchesGoldenFile].
+  const _BufferGoldenMatcher(this.key, this.version);
+
+  /// The [key] to the golden image.
+  final Uri key;
+
+  /// The [version] of the golden image.
+  final int version;
+
+  @override
+  Future<String> matchAsync(dynamic item) async {
+    Uint8List buffer;
+    if (item is List<int>) {
+      buffer = Uint8List.fromList(item);
+    } else if (item is Future<List<int>>) {
+      buffer = Uint8List.fromList(await item);
+    } else {
+      throw 'Expected `List<int>` or `Future<List<int>>`, instead found: ${item.runtimeType}';
+    }
+    final Uri testNameUri = goldenFileComparator.getTestUri(key, version);
+    if (autoUpdateGoldenFiles) {
+      await goldenFileComparator.update(testNameUri, buffer);
+      return null;
+    }
+    try {
+      final bool success = await goldenFileComparator.compare(buffer, testNameUri);
```

Comment:
You mentioned handling this manually until the client is ready for luci testing. It looks like in post-submit on luci this comparison is a no-op: https://github.com/flutter/flutter/blob/0be9127dfd934b8fe4bb363ea47d895d1e9830f9/packages/flutter_goldens/lib/flutter_goldens.dart#L489

Reply:
Yep. I can definitely follow up with that based on the discussion on #49749.

Reply:
Ok. Is this an already existing test? If so, idk if changing it to a no-op test in the interim is a good idea.